doc/changelog.md

```diff
@@ -15,6 +15,7 @@ To be released at some future point in time
 
 Description
 
+- Add type check to model params
```

Comment:
Make these more descriptive (you're doing more than just type checking):

```suggestion
Allow specifying Model and Ensemble parameters with number-like types (e.g. numpy types)
```

Reply:
changed